src/main/java/firsttask/taskmanager/Controller/TaskController.java

```diff
@@ -94,6 +100,23 @@ Task edTask(@RequestBody Task editTask, @PathVariable Long id) throws AccessDeni
 
     }
 
+    private void checkDate(Task task) {
+        Date startDateToCheck=task.getStartDate();
+        Date endDateToCheck=task.getEndDate();
+
+        List<Task> tasks=taskRepository.findAll();
```

Comment:
Do you get all tasks here? or all tasks for a specific user?

Reply:
its suppose to be for one user, my bad  ill change it now sry!